networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -139,13 +139,112 @@ include::modules/nw-multus-tap-object.adoc[leveloffset=+2]
 
 * For more information about enabling an SELinux boolean on a node, see xref:../../nodes/nodes/nodes-nodes-managing.adoc#nodes-nodes-working-setting-booleans_nodes-nodes-managing[Setting SELinux booleans]
 
+// Set of includes pertains to OVN-Kubernetes additional network
 include::modules/configuring-ovnk-additional-networks.adoc[leveloffset=+2]
+
+[id="{context}_configuration-additional-network-types-supported-platforms"]
+==== Supported platforms for OVN-Kubernetes additional network
+
+You can use an OVN-Kubernetes additional network with the following supported platforms:
+
+- Bare metal
+- {ibmpowerProductName}
+- {ibmzProductName}
+- {linuxoneProductName}
+- VMware vSphere
+- {rh-openstack-first}
+
+// Workaround lack of xref in modules
+[id="{context}_configuration-additional-network-types-prerequisites"]
+==== Prerequisites for configuring OVN-Kubernetes additional network
+
+- Install NMstate Operator. For more information, see xref:../../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[About the Kubernetes NMState Operator]
+
+[id="{context}_configuration-additional-network-interface"]
+==== Configuration for an OVN-Kubernetes additional network mapping
```

Comment:
these sections are only required for the localnet topology. Meaning `layer2` topologies do not require any of this.

I think the user can misunderstand and think this is always required.